### PR TITLE
Add postmortem for 2019-02-05 partial outage.

### PR DESCRIPTION
**Pull request by Katharine**
Haven't updated the index to avoid a conflict with #10911, which has not yet merged.

### CODE REVIEW

**Comment by BenTheElder** [comment]
Thank you! we should merge #10911

File: docs/post-mortems/2019-02-05.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+- Breaking something that approves and then deploys itself makes rolling back trickier.
+- The strong ordering requirement (branchprotector must be deployed at least four hours before anything else)
+  added complexity to the rollout, which failed.
+- PRs that are missing "optional" statuses did not automatically recover after the outage and must
+  manually kicked to recover them.
+- Merging multiple major changes simultaneously is less than ideal.
+
+### Where we got lucky
+
+- Apparently nobody was watching their PRs very closely around 9am PST, so nobody noticed.
```
Comment (stevekuznetsov) [inline docs/post-mortems/2019-02-05.md]: If a Prow falls in a forest ...

File: docs/post-mortems/2019-02-05.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+what is your PodNamespace in the config?
+>
+>Katharine [10:32 AM]  
+`test-pods`: https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L41
+>
+>stevekuznetsov [10:32 AM]  
+very interesting  
+oh  
+lmao  
+:facepalm:
```
Comment (stevekuznetsov) [inline docs/post-mortems/2019-02-05.md]: I am happy this is commited to `git` history forever

**Comment by mithrav** [comment]
Ty for the writeup Katharine!

**Pull request merged**